### PR TITLE
#Ft Email Verification on Signup

### PR DESCRIPTION
**Pull request by dreplica**
Screenshot_20200405-202923_Expo

#### What does this PR do?
provides user access to verify their email address

Have an overview feature of the app displayed in home screen.
- Shows email and OTP input field
- Buttons for Resending verification or confirming with SMS

#### Description of Task to be completed?
- used an OTP library to display OTP Input.

Have the home screen show the following:

1. a text showing welcome to the app.
2. And a clickable button with no action attached to it.

#### How should this be manually tested?

After cloning the repo,
`CD` into it and `RUN`

javascript
git checkout develop

git pull origin develop

yarn install

yarn test

Using an emulator: Launch the app, and you should see the home screen after loading.

### CODE REVIEW

**Comment by ogbodo** [comment]
Based on that I can see in the screen shot attached to the PR you made, you got the whole concept right and positioning of elements. But you need to work on the font all together (like the font size, family and font weight), as well as the OTP and email inputFields. PLEASE REFERE TO THE DESIGN GIVEN TO YOU AND COMPARE WITH WHAT WE HAVE HERE TO CONFIRM WHAT AM SAYING, OK.

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,8 @@ web-build/
 web-report/
 .env
 yarn.lock
+./screens/signup/verifyStyles
+./screens/signup/verifyAccount
```
Comment (ogbodo) [inline .gitignore:15]: Please could you explain what you want to achieve on line 14 and 15?

File: src/screens/signup/emailVerify.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+export default function VerifyAccount() {
+  const { colors } = useThemeContext();
+  const [inputText, setInputText] = useState("")
+  return (
+    <Container>
+      <ScrollView showsVerticalScrollIndicator={false}>
+        <Pagination>3/3</Pagination>
+        <Header>
+          <Text style={{ width: 100, fontSize: 20 }}>Lastly,</Text>
```
Comment (ogbodo) [inline src/screens/signup/emailVerify.tsx:23]: Please you can make this a styled component for uniformity and to make the code slim as much as possible.

File: src/screens/signup/emailVerify.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+          <Text
```
Comment (ogbodo) [inline src/screens/signup/emailVerify.tsx:24]: Please you can make this a styled component for uniformity and to make the code slim as much as possible.

File: src/screens/signup/emailVerify.tsx [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+              fontSize: 35,
+              color: colors.BG_YELLOW_COLOR,
+              marginTop: 10
+            }}
+          >
+            Verify your Account
+          </Text>
+        </Header>
+        <Input 
+        placeholder={'enter email'}
```
Comment (ogbodo) [inline src/screens/signup/emailVerify.tsx:36]:
```suggestion
        placeholder='enter email'
```

File: src/screens/signup/emailVerify.tsx [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+          buttonStyle={{
+            width: 380,
+            elevation: 0,
+            height: 60
+          }}
+          textStyle={{
+            fontSize: 20
+          }}
+        />
+        <Text
```
Comment (ogbodo) [inline src/screens/signup/emailVerify.tsx:93]: Please you can make this a styled component for uniformity and to make the code slim as much as possible.

**Comment by ogbodo** [comment]
Please similar correction should be applied to other parts of the code where necessary.

File: .gitignore (hunk 1 above)
Comment (dreplica) [reply .gitignore:15]: I forgot to remove it. thanks

**Comment by ogbodo** [comment]
Good job

**Pull request merged**